index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ieProduct Landing Page=edge">
+    <title>Fresh Premium Organic Vegetables</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  
+  <body>
+    
+   <div>
+    <header id="header"><div><img id = "header-img" src=""><div>FreshOs</div></div>
+        
+        <nav id="nav-bar">
+            <div class="nav-list">
+                <div >
+                    <a class="nav-link" href="#productDetails">Product details</a>
+                </div>
+                <div >
+                    <a class="nav-link" href="#HowItWorks"> How it Works </a>
+                </div>
+                <div >
+                    <a class="nav-link" href="#Pricing"> Pricing</a>
+                </div>
+            </div>
+        </nav>
+        </header>
+   </div>
+   <div>
+    <main>
```

Comment:
- [ ] I think this comment is vague and the purpose of this comment is not very clear to the student. as `main` is a semantic HTML element, I think it would be better to direct this inline-comment towards the `div` that is the direct parent of `main` at line `31`. as `main` is one of the semantic HTML element and taking a `main` inside a `div` is not a very good use of semantics, kindly consider directing this inline-comment to the `div` at line `31`.